app.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const app = express();
+const {getCategories} = require('./controllers/categories-controllers');
+
+app.use(express.json());
+
+app.get('/api/categories', getCategories);
+
+// error handling
+
+// psql errors
+app.use((err, req, res, next) => {
+    if (err.code === '22P02') {
+        res.status(400).send('baaad request x');
+    } else {
+        next(err);
+    }
+})
+// custom errors
+app.use((err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.status(err.status).send({msg: err.msg});
+    } else {
+        next(err);
+    }
+})
```

Comment:
Error handlers look great but technically irrelevant at this stage. If there are no tests related to bad requests or custom errors (which there won't be for this endpoint) they shouldn't be here. Please remove and reintroduce for a later ticket when they are first required